include/identity.php

```diff
@@ -374,7 +374,10 @@ function profile_sidebar($profile, $block = 0) {
 			if (dbm::is_result($r))
 				$updated =  date("c", strtotime($r[0]['updated']));
 
-			$r = q("SELECT COUNT(*) AS `total` FROM `contact` WHERE `uid` = %d AND NOT `self` AND NOT `blocked` AND NOT `hidden` AND NOT `archive`
+			$r = q("SELECT COUNT(*) AS `total` FROM `contact`
```

Comment:
Nice to indent the SQL statement so it becomes more readable by humans (not maschines, of course).